zep_python/memory/client.py

```diff
@@ -41,29 +39,6 @@ def __init__(self, aclient: httpx.AsyncClient, client: httpx.Client) -> None:
         self.aclient = aclient
         self.client = client
 
-    def _parse_get_memory_response(self, response_data: Any) -> Memory:
```

Comment:
why did you remove this method?

Reply:
It didn't appear to be doing anything more than pydantic does when parsing data into a model. I can't recall why we needed it to begin with :-/